src/main/java/ch/uzh/ifi/hase/soprafs21/service/WebSocketService.java

```diff
@@ -40,6 +41,11 @@
 
     @Autowired
     public SimpMessagingTemplate simp;
+    private final UserService userService;
+
+    public WebSocketService(UserService userService) {
```

Comment:
@Autowired
 public UserService userService;

sollte reichen, nicht?

Reply:
wurde in allen anderen service klassen und controllers ebenfalls so gemacht, @Autowired Userservice wäre das erste und einzige mal hier so gemacht worden.
Kann ich machen, sehe allerdings nicht wirklich wieso wir es hier anders machen sollten.
Gib mir Bescheid wenn du es gerne Autowired haben möchtest.

Reply:
okok, ja dann lassen wir es mal so